multi_agent_controller:
  ros__parameters:
    kpO: 35.0
    kiO: 0.0
    kdO: 0.0
    v0: 10.0
    alpha: 6.0
    log_hz: 1.0
    meta_assignments:
      Meta_1: [3,4,6]
      Meta_2: [7,8,9]
      Meta_3: [11,12,13]
      Meta_4: []
      Meta_5: []
